Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/detail/AgendaDetailFragment.java

```diff
@@ -90,7 +90,15 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         Bundle bundle = getArguments();
         final ScheduleRow itemData = gson.fromJson(bundle.getString(Schedule.SCHEDULE_ITEM_ROW), ScheduleRow.class);
 
-        final String speakerName = itemData.speakerName;
+        final String speakerName;
```

Comment:
sorry..I should have renamed this primarySpeakerName and created an array of speaker names...if you prefer, I can do this instead and that'll fix your speaker page so you don't have to do this?
p.s., you should probably run the speaker description fields through the html converter before displaying it